config/command_handler.yml

```diff
@@ -31,5 +31,6 @@ services:
       - '@Mollie\Service\MollieOrderCreationService'
       - '@mollie.link'
       - '@mollie'
+      - '@Mollie\Handler\RetryHandler'
```

Comment:
Perfect place to have interface RetryProcessHandlerInterface which would return the RetryHandler for easier extendability, not required though